Add user documentation for KEP-2937 to website (#3175)
kubernetes-sigs/kueue: site/content/en/docs/tasks/manage/administer_cluster_quotas.md

  excludeResourcePrefixes:
  - "example.com"
```

## Transform resources for quota management

An administrator may customize how the resources requested by Pods are
converted into Workload resource requests. This enables
the admission and quota calculations done by ClusterQueues to be performed
on a transformed set of resources requirements without changing
the resource requests and limits that the Pods created by the Workload
will present to the Kubernetes Scheduler when the Workload is admitted by
a ClusterQueue. Customizations are defined by specifying resource transformations
in the Kueue configuration as a cluster-level setting.

{{< feature-state state="alpha" for_version="v0.9" >}}
{{% alert title="Note" color="primary" %}}

> `transformations` is a Alpha feature that is not enabled by default.

Reviewer: ```suggestion
`transformations` is an Alpha feature that is not enabled by default.
```

output resources by multiplying the input resource quantity by a scaling factor.
The input resource may either be retained (default) or removed from the transformed resources. If no transformation is defined for an input resource, it is retained without change.

After enabling the feature gate, follow the [installation instructions for using a custom configuration](/docs/installation#install-a-custom-configured-released-version)
and extend the Kueue configuration with fields similar to the following:

```yaml
apiVersion: config.kueue.x-k8s.io/v1beta1
kind: Configuration
resources:
  transformations:
  - input: example.com/gpu-type1:
    strategy: Replace
    outputs:
      example.com/gpu-memory: 5Gi
      example.com/credits: 10
  - input: example.com/gpu-type2:
    strategy: Replace
    outputs:
      example.com/gpu-memory: 20Gi
      example.com/credits: 40
  - input: cpu
    strategy: Retain
    outputs:
      example.com/credits: 1
```

With this example configuration, a Pod that requests:
```yaml
    resources:
      requests:
        cpu: 1
        memory: 100Gi
      limits:
        example.com/gpu-type1: 2
        example.com/gpu-type2: 1
```

> will for quota purposes have an effective resource request of:

Reviewer: ```suggestion

The Workload obtains an effective resource requests for quota purposes:

```
